linux/library/src/internal/text_input_plugin.cc

```diff
@@ -170,4 +175,12 @@ void TextInputPlugin::SendStateUpdate(const TextInputModel &model) {
   InvokeMethod(kUpdateEditingStateMethod, model.GetState());
 }
 
+void TextInputPlugin::EnterPressed(const TextInputModel &model) {
+  Json::Value args = Json::arrayValue;
+  args.append(model.client_id());
+  args.append(kNewLineAction);
```

Comment:
Why the inconsistency with the macOS patch's behavior of currently using Done instead of NewLine?

Reply:
Changed to Done to align with macOS change